src/main/java/com/hubspot/jinjava/interpret/Context.java

```diff
@@ -94,25 +95,38 @@ public Context setValidationMode(boolean validationMode) {
   private boolean validationMode = false;
 
   public Context() {
-    this(null, null, null);
+    this(null, null, null, false);
+  }
+
+  public Context(boolean global) {
```

Comment:
Is this constructor necessary? I think you could just use one of the others and pass in nulls. I find it confusing when a method with one parameter could be take two types of arguments.